### PR TITLE
Adding reviews

### PR DESCRIPTION
**Pull request by voycik**

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -12,10 +12,12 @@ def new
   end
 
   def create
-    @review = Review.new(review_params)
+
```
Comment (MarekParafianowicz) [inline app/controllers/reviews_controller.rb]: Obsolete empty lines - check whole PR please

File: app/views/places/show.html.haml [hunk 2]
```diff
@@ -32,4 +32,28 @@
...
+
+
+%h4 Add new review for  this place
+
+.content
+  = render partial: 'reviews/reviews', place: @place
+
+
+
+  %table.table.table-hover
```
Comment (MarekParafianowicz) [inline app/views/places/show.html.haml]: Cade related to reviews can be moved to reviews partial (like the form for reviews)

File: app/views/places/show.html.haml [hunk 3]
```diff
@@ -32,4 +32,28 @@
...
+    %thead
+      %tr
+        %th{:scope => "col"} #
```
Comment (MarekParafianowicz) [inline app/views/places/show.html.haml]: Old hash syntax

File: app/views/places/show.html.haml [hunk 4]
```diff
@@ -32,4 +32,28 @@
...
+        %th{:scope => "col"} Author
+        %th{:scope => "col"} Review
+        %th{:scope => "col"} Rate
+    %tbody
+      - Review.all.each_with_index do |review, index|
```
Comment (MarekParafianowicz) [inline app/views/places/show.html.haml]: You are rendering all reviews not only related to this place

File: config/routes.rb [hunk 5]
```diff
@@ -3,6 +3,8 @@
 
   root 'places#index'
 
-  resources :reviews
-  resources :places
+  # resources :reviews
```
Comment (MarekParafianowicz) [inline config/routes.rb]: Delete this line

File: app/views/places/show.html.haml (hunk 3 above)
Comment (voycik) [reply app/views/places/show.html.haml]: I'll change that

File: app/views/places/show.html.haml (hunk 4 above)
Comment (voycik) [reply app/views/places/show.html.haml]: Thank, I've changed it

File: config/routes.rb (hunk 5 above)
Comment (voycik) [reply config/routes.rb]: Ready

**Pull request merged**